project3/styles.css

```diff
@@ -0,0 +1,321 @@
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #163346;
+}
+
+form {
+    background-color: #1e2019;
+    max-width: 70%;
+    margin-right: auto;
+    margin-left: auto;
+    padding: 20px;
+
+    display: block;
+    position: relative;
+    bottom: 50px;
+}
+
+label {
+    font-family: "Benchnine", monospace;
+    font-size: 22px;
+    color: #cececd;
+}
+
+
+legend {
+    font-family: "Benchnine", monospace;
+    font-size: 22px;
+    color: #cececd;
+}
+
+.sr-only {
+    border: 0;
+    clip: rect(0 0 0 0);
+    overflow: hidden;
+    margin: -1px;
+    padding: 0;
+    position: absolute !important;
+    height: 1px;
+    width: 1px;
+}
+
+section {
+    background-color: #1e2019;
+    margin-right: auto;
+    margin-left: auto;
+    max-width: 70%;
+}
+
+.input,
+.textarea,
+.select {
+    font-family: inherit;
+    font-size: inherit;
+}
+
+
+.age {
+    padding-bottom: 30px;
+}
+
+fieldset {
+    border: 0px;
+    padding: 0px;
+    margin-right: auto;
+    margin-left: auto;
+    max-width: 70%;
+    display: inline-block;
```

Comment:
I would suggest that this rule belongs in a media query and not here; at the smallest size, these `fieldset`s are single column and not really needed to be `inline-block` -- that change comes into play when they display in columns.